rocky/reports/report_types/aggregate_organisation_report/introduction.html

```diff
@@ -8,18 +8,20 @@ <h1>{{ report_name }}</h1>
                 <button aria-controls="export-add" class="dropdown-button ghost">
                     {% translate "Export" %}<span aria-hidden="true" class="icon ti-chevron-down"></span>
                 </button>
-                <ul id="export-add" aria-expanded="false" class="dropdown-list">
-                    <li>
-                        <a href="{{ report_download_pdf_url }}"
-                           target="_blank"
-                           rel="noopener noreferrer">{% translate "Download PDF" %}</a>
-                    </li>
-                    <li>
-                        <a href="{{ report_download_json_url }}"
-                           target="_blank"
-                           rel="noopener noreferrer">{% translate "Download JSON" %}</a>
-                    </li>
-                </ul>
+                <div id="export-add" aria-expanded="false" class="dropdown-list">
+                    <ul>
+                        <li>
+                            <a href="{{ report_download_pdf_url }}"
+                               target="_blank"
+                               rel="noopener noreferrer">{% translate "Download PDF" %}</a>
```

Comment:
rel="noopener noreferrer" are not needed on internal / local links. They are not problematic either.

Reply:
Ah yes, I will remove it as it's not needed.